scm-ui/ui-styles/src/scm.scss

```diff
@@ -160,6 +160,10 @@ $danger-25: scale-color($danger, $lightness: 75%);
   color: $blue-light !important;
 }
 
+.has-text-blue {
+  color: $blue !important;
+}
+
```

Comment:
How does it differ from `has-text-info` or `has-text-link`?

Reply:
In no way -- learned something...